### DISCUSSION TITLE
In which desert - the hottest in the world - are the Tibesti Mountains located?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#317] [Galaxy Trivia] [answered]
**Hint:** "The greatest desert"

**Comment by thuongtruong1009** [comment, marked as answer]
The Tibesti Mountains are located in the Sahara Desert, which is the hottest desert in the world. Specifically, the Tibesti Mountains are situated in the central part of the Sahara, straddling the border between Chad and Libya.
